samples/Nancy.Swagger.Demo/Nancy.Swagger.Demo.csproj

```diff
@@ -126,6 +128,13 @@
   <PropertyGroup>
     <PreBuildEvent>xcopy "$(SolutionDir)lib\swagger-ui\dist" "$(ProjectDir)swagger-ui" /S /I /Y</PreBuildEvent>
   </PropertyGroup>
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('$(SolutionDir)\.nuget\NuGet.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\.nuget\NuGet.targets'))" />
```

Comment:
This is a problem. The old-style package restore has been enabled, and that's not something that we want :cry: To quote @davidebbo:

> Every Time someone enables nuget package restore on a solution, a kitten dies. Learn the new workflow!

http://blog.davidebbo.com/2014/01/the-right-way-to-restore-nuget-packages.html